src/pages/Editor.vue

```diff
@@ -297,20 +297,28 @@ export default {
       const patStr = this.drawingTool.toString();
       this.qrCode = patStr;
     },
-    onQROpen: function () {
+    onQROpen: function() {
       const patStr = this.drawingTool.toString();
       this.qrCode = patStr;
     },
     loadFromExample(exampleNumber) {
       let currentExample = examples[exampleNumber];
       this.searchResult = currentExample;
       this.$set(this.iiif, "url", currentExample.large_iiif_url);
-      this.$refs["imageloader"].setCropData(currentExample.crop);
+      let self = this;
+      // wait a tiny big before loading the cropper
+      setTimeout(function() {
+        self.$refs["imageloader"].setCropData(currentExample.crop);
+      }, 1000);
```

Comment:
A second seems long here.  Maybe a bit shorter?  200ms?

Reply:
I wasn't seeing any appreciable wait, so I left it, but I'm not seeing the error at 200ms either. adjusted